src/views/settings/Security/Auth.vue

```diff
@@ -31,6 +31,7 @@ export default {
             [
               'SECURITY_MFA_AUTH',
               'SECURITY_MFA_IN_LOGIN_PAGE',
+              'SECURITY_MFA_BY_EMAIL',
               'SECURITY_MFA_AUTH_ENABLED_FOR_THIRD_PARTY',
               'SECURITY_MFA_VERIFY_TTL',
               'OTP_ISSUER_NAME', 'OTP_VALID_WINDOW'
```

Comment:
The code seems fine with no apparent errors or issues to optimize. Here's the detailed feedback:

1. `export default { ... };` is missing at the beginning of the object declaration.
   
To fix it:

```
export default {
...
};
```